src/Main.java

```diff
@@ -2,7 +2,135 @@
+
+Напишите программу, которая определяет, чем пользуется клиент (iOS или Android),
+и выдает соответствующее сообщение:
+
+Для iOS — «Установите версию приложения для iOS по ссылке».
+Для Android — «Установите версию приложения для Android по ссылке».
+Объявите переменную clientOS, которая равна 0 или 1 (0 — iOS, 1 — Android).*/
+        int clientOS=1;
+        int OSForIOS=0;
+        int OSForAndroid=1;
+        if(clientOS==OSForIOS){
+            System.out.println("Установите версию приложения для iOS по ссылке...");
+        }
+        else {
+            System.out.println("Установите версию приложения для Android по ссылке...");
+        }
+        System.out.println("===============================");
+/*Задача 2
+Усложним предыдущую задачу. Теперь нам нужно знать не только операционную систему телефона,
+но и год его создания.
+Ваша задача — написать программу,
+которая выдает соответствующее сообщение клиенту при наличии двух условий.
+Если год выпуска ранее 2015 года, то к сообщению об установке нужно добавить информацию об облегченной версии:
+Для iOS оно будет звучать так: «Установите облегченную версию приложения для iOS по ссылке».
+Для Android: «Установите облегченную версию приложения для Android по ссылке».
+Для пользователей телефонов 2015 года выпуска и позже нужно вывести обычное предложение об установке приложения.
+Для года создания телефона используйте переменную clientDeviceYear, в которой необходимо указать 2015 год.*/
+
+        int clientDeviseYear=2015;
+        int yearPhone=2012;
+        if(clientOS==OSForIOS&&yearPhone<clientDeviseYear){
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке...");
+        } else if (clientOS==OSForIOS&&yearPhone>=clientDeviseYear) {
+            System.out.println("Установите версию приложения для iOS по ссылке...");
+        } else if (clientOS==OSForAndroid&&yearPhone<clientDeviseYear) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке...");
+        }else {
+            System.out.println("Установите версию приложения для Android по ссылке...");
+        }
+        System.out.println("==============================");
+/*Задача 3
+Напишите программу, которая определяет, является ли год високосным или нет.
+
+Переменную года назовите year, в которую можно подставить значение интересующего нас года. Например, 2021.
+
+Программа должна определять, високосный год или нет, и выводить соответствующее сообщение:
+« …. год является високосным» или «... год не является високосным».
+
+Небольшая справка: високосным является каждый четвертый год,
+но не является каждый сотый. Также високосным является каждый четырехсотый год.
+Год должен быть больше, чем 1584 (в котором был введен високосный год).*/
+        int year=2027;
+        int startLeapYear=1586;
+        int missedYears=4;
+        if(year>=startLeapYear&&year%4==0&&year%100!=0) {
```

Comment:
```suggestion
        if(year >= startLeapYear && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```
Вот такое должно быть условие